src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -123,6 +125,10 @@ export const ConsentGroupReview = hh(class ConsentGroupReview extends Component
     this.init();
   }
 
+  componentWillUnmount() {
+    spinnerService._unregister(CONSENT_GROUP_REVIEW_SPINNER);
+  }
```

Comment:
I'm not liking this pattern of requiring that every component un-register itself. It's boilerplate that any new components will need to copy/paste. Would be much better if we could use a HOC for this and wrap the components with a spinner HOC.